src/manifest.json

```diff
@@ -5,16 +5,11 @@
     "manifest_version": 2,
     "content_scripts": [{
         "js": [
-            "vendor/jquery-2.1.4.min.js",
-            "ChromeUserNameCache.js",
-            "ReplaceRestricter.js",
-            "UserIdStringReplacer.js",
-            "UserIdReplacer.js",
-            "index.js"
+            "index.bundle.js"
         ],
         "matches": [
-            "https://github.wdf.sap.corp/*",
-            "https://huboard.mo.sap.corp/*"
+            "*://github.wdf.sap.corp/*",

```

Comment:
Changing or adapting the matches config will disable the chrome extension for all currently installed users.